lib/rules/no-duplicate-selectors/README.md

```diff
@@ -100,3 +100,22 @@ a {
   & c {}
 }
 ```
+
+## Optional secondary options
+
+### `disallowInList: true | false` (default: `false`)
+
+This option will dissallow duplication selectors within a group selectors
+
+For example, with `true`, the following patterns are considered violations:
```

Comment:
```suggestion
For example, with `true`.

The following patterns are considered violations:
```

It's a bit clunky, but the sentence needs to split for the color highlighting on the website to work, [for example](https://stylelint.io/user-guide/rules/selector-class-pattern/#optional-secondary-options).